pkg/sql/parser/parse.go

```diff
@@ -152,13 +152,23 @@ func (p *Parser) scanOneStmt() (sql string, tokens []sqlSymType, done bool) {
 	// We make the resulting token positions match the returned string.
 	lval.pos = 0
 	tokens = append(tokens, lval)
+	var preValID int32
+	curFuncBodyCnt := 0
```

Comment:
Add some commentary to indicate what this is about?

Reply:
added.